catalog/datasets/rkdartists.jsonld

```diff
@@ -8,6 +8,12 @@
       "@value": "RKDartists"
     }
   ],
+  "alternateName": [
+    {
+      "@language": "en",
+      "@value": "RKD"
```

Comment:
I decided to provide `alternateName` with a language (which isn’t really used yet), just like `name`. For RKD this is a bit weird: while RKD is a Dutch abbreviation, as an abbreviation for RKDArtists we could consider it English.

While abbreviations _can_ be language-specific (e.g. English UN vs. Dutch VN), we could just consider the abbreviations universal labels.

Reply:
Agree! However, should `RKD` be `RKDartists` (the name of the dataset, not the name of the organization)?